docassemble/AssemblyLine/al_document.py

```diff
@@ -1393,25 +1419,38 @@ def ocr_pages(self):
 
     def as_pdf(
         self,
-        prefix="",
+        *,
+        refresh: bool = False,
+        bates_prefix: str = "",
         pdfa: bool = False,
         add_page_numbers: bool = True,
         add_cover_page: bool = True,
         filename: str = None,
     ) -> DAFile:
-        if hasattr(self._cache, "_file"):
-            return self._cache._file
```

Comment:
This code was used to make sure the file was always fresh after an edit button is clicked. I can see that might not be compatible with OCR in the background though. Any thoughts on how to handle that need?